### DISCUSSION TITLE
Does the app export messages in Recently Deleted?

### DISCUSSION THREAD

**Discussion by miles267** [ReagentX/imessage-exporter#256] [Q&A] [answered]
Hi - can you please confirm whether iMessage-exporter will export message threads in the Recently Deleted area of iMessage? Or must these messages first be recovered before running the export? My hope is that it will include this data while it still resides in the Recently Deleted. Please confirm. Thank you.

**Comment by ReagentX** [comment, marked as answer]
Recently Deleted messages are automatically included; this is addressed in the [FAQ](https://github.com/ReagentX/imessage-exporter/blob/develop/docs/faq.md#are-messages-deleted-from-the-messages-app-erased-from-the-database) document.
